content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,75 @@
-| --- | --- |
-| `adapter`| Specification of Istio extension declarations |
-| `attributemanifest` | Specification of Istio extension declarations |
-| `template` | Specification of Istio extension declarations |
-| `handler` | Specification of extension invocations |
-| `rule` | Specification of extension invocations |
-| `instance` | Specification of extension invocations |
-
-In the event you are using the removed mixer configuration schemas, set
-the following Helm flags during upgrade of the main Helm chart:
-`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
+to the [1.3 release notes](/about/notes/1.3/).
+
+## Installation and upgrade
+
+We simplified the configuration model for Mixer and removed support for
+adapter-specific and template-specific Custom Resource Definitions (CRDs)
+entirely in 1.3. Please move to the new configuration model.
+
+We removed the Mixer CRDs from the system to simplify the configuration
+model, improve Mixer's performance in Kubernetes deployments, and improve
+reliability in various Kubernetes environments.
+
+## Traffic management
+
+Istio now captures all ports by default. If your services listen on `127.0.0.1`,
+you probably don't specify container ports due to the following reasons:
+
+- To avoid exposing your service.
+- To intentionally bypass Envoy.
+
+If that is the case, you must opt out of port capturing with the
+`traffic.sidecar.istio.io/excludeInboundPorts` option.
+
+{{< warning >}}
+This change exposes any applications listening on `127.0.0.1`
+since Envoy connects over `localhost`. Opt out to avoid exposing such applications.
+{{< /warning >}}
+
+Protocol sniffing is now enabled by default. Disable port sniffing with the
+`--set pilot.enableProtocolSniffing=false` option when you upgrade to get the
+previous behavior. To learn more see our [protocol selection page](/docs/ops/traffic-management/protocol-selection/).
+
+To specify a hostname in multiple namespaces, you must select a single host using
+a [`Sidecar` resource](/docs/reference/config/networking/v1alpha3/sidecar/).
+
+## Trust domain validation
+
+Trust domain validation is new in Istio 1.3. If you only have one trust domain
+or you don't enable mutual TLS through authentication policies, there is nothing
+you must do.
+
+To opt-out the trust domain validation, include the following flag in your Helm
+template before upgrading to Istio 1.3:
+`--set pilot.env.PILOT_SKIP_VALIDATE_TRUST_DOMAIN=true`
+
+## Secret Discovery Service
+
+In Istio 1.3, we are taking advantage of improvements in Kubernetes to issue
+certificates for workload instances more securely.
+
+Kubernetes 1.12 introduces `trustworthy` JWTs to solve these issues.
+[Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md)
+introduced the ability to change the value of the `aud` field to a value other
+than the API server. The `aud` field represents the audience in Kubernetes. To
```

Comment:
This sentence "The `aud` field represents the audience in Kubernetes" is not technically correct, and I think you can remove it. If not, you can change it to something like "The `aud` field represents the audience of the trustworthy JWTs, which is the Istio CA"

Reply:
cc @myidpt